microros_utils/library_builder.py

```diff
@@ -194,7 +194,7 @@ def package_mcu_library(self):
         shutil.copytree(self.build_folder + "/mcu/install/include", self.includes)
 
         # Fix include paths
-        if self.distro is not "galactic":
```

Comment:
I have doubts about this. These sections should be executed for `rolling` and upcoming versions, so if we set `self.distro != "galactic"` it will be executed for Foxy and it is not correct.

We need something like `self.distro > galactic`. Maybe we should add some version numbers somewhere?
Or maybe we can just say `self.distro != "galactic" and self.distro != "foxy"`?

Reply:
quickfix here f6435fea16268e90cfdc866b2a65f7f3c05b4762